OpenRA.Mods.Cnc/Traits/Disguise.cs

```diff
@@ -76,13 +78,21 @@ class DisguiseInfo : ITraitInfo
 		[Desc("The condition to grant to self while disguised.")]
 		public readonly string DisguisedCondition = null;
 
-		[Desc("Triggers which cause the actor to drop it's disguise. Possible values: None, Attack, Damaged.")]
+		// Added this for more control over when disguises break
+		[Desc("Events leading to the actor breaking Disguise. Possible values are: Attack, Move, Demolish, and Damaged.")]
```

Comment:
None is valid and used - asin having the enum on the value None alone will mean it won't drop the disguise in any event. That's how disguise worked in RA2. None is the sole reason I added this enum back in the PR this was introduced.